packages/design-system/README.md

```diff
@@ -60,7 +60,8 @@ Limit changes to styled-components scope.
 
 ### End-to-End tests
 
-Prior to end-to-end tests if you want to test interactions in real browsers.
+Simple renders will be covered by Chromatic.
+Prior to end-to-end tests, using Cypress, if you have to test interactions in real browsers.
```

Comment:
```suggestion
Prior to end-to-end tests, using Cypress, if you have to test interactions in real browsers.
```
This doesn't make grammatical sense.

Reply:
+1, don't understand what you want to say here.
The rest LGTM

Reply:
+1, don't understand what you want to say here.
The rest LGTM